include/alpaka/math/MathGenericSycl.hpp

```diff
@@ -1,4 +1,4 @@
-/* Copyright 2022 Jan Stephan
+/* Copyright 2023 Jan Stephan, Sergei Bastrakov, René Widera, Luca Ferragina, Andrea Bocci
```

Comment:
Can we put these changes into an own PR?

Reply:
Not really, as they are needed to make the tests pass.